mkdocs.yml

```diff
@@ -68,6 +68,7 @@ nav:
     - Kubecost: 'addons/kubecost.md'
     - Kubeflow: 'addons/kubeflow.md'
     - KubeRay Operator: 'addons/kuberay-operator.md'
+    - Kubernetes Ingress: 'addons/kubernetes-ingress.md'
```

Comment:
Kubernetes ingress is a very broad definition. We have to be specific that it is nginx as there are many kubernetes ingress controllers. AWS ALB ingress controller is one of the kubernetes ingress controllers. 

I see you made changes to abbreviate. I suggest the following naming to make it clear to the customers (using standard nomenclature):

https://github.com/kubernetes/ingress-nginx - is the repo and project name. 

Let's rename to ingress-nginx throughout. The addon name will be ingress-nginx and the entries in the doc will be the same. We will specify that the difference between ingress-nginx and nginx addon are that the former is upstream kubernetes nginx and the latter is F5.